calculate_documentation_coverage.py

```diff
@@ -44,7 +44,7 @@ def parse_files(directory):
     return public_methods, documented_methods
 
 # Specify the directory containing your Python files
-directory = "../src/qudi/"
+directory = "src/qudi"
```

Comment:
Would suggest using os.path.join since different operating systems use different slash directions (forward vs backward) to specify directories.

Reply:
I already change it, so have a look on it and approve if it looks good!